source/_posts/2018-01-03-using-git-hooks-to-improve-your-day-to-day-workflow.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Using git hooks to improve your day-to-day workflow"
+category: git
+comments: true
+author:
+  name: Federico Kauffman
+  email: federico@wyeworks.com
+  twitter_handle: fedekauffman
+  github_handle: fedekau
+  image: /images/team/federico-kauffman.jpg
+  description: Software Engineer at Wyeworks. Currently working with Javascript and Ruby. Learnaholic.
+---
+
+If you have been developing software for some time you have probably noticed that there are lot of things that can go wrong, no matter how hard you try there is always something you might forget because after all, we are just humans doing an extremly difficult task: Telling a computer what to do.
+
+In this post I will expose some of the **git-hooks** we use in some projects here at Wyeworks to make developers life easier by preventing bad commits to even leave their computers. I will cover *linting*, *tests* and *commit formating* use cases.
+
+<!--more-->
+
+## What are git-hooks?
+
+Like many other Version Control Systems, Git has a way to fire off custom scripts when certain important actions occur. There are two groups of these hooks: *client-side* and *server-side*. Client-side hooks are triggered by operations such as committing and merging, while server-side hooks run on network operations such as receiving pushed commits. You can use these hooks for all sorts of reasons.
```

Comment:
`There are two groups of these hooks:` -> `These hooks can be categorized in two groups:`

Reply:
Will leave as is because I am quoting the documentation